frontend/src/tbpro/elements/TextInput.vue

```diff
@@ -61,20 +72,23 @@ const onChange = () => {
       <slot/>
       <span v-if="required && model?.length === 0" class="required">*</span>
     </span>
-    <input
-      class="tbpro-input"
-      v-model="model"
-      :class="{'dirty': isDirty}"
-      :type="type"
-      :id="name"
-      :name="name"
-      :disabled="disabled"
-      :placeholder="placeholder"
-      :required="required"
-      @invalid="onInvalid"
-      @change="onChange"
-      ref="inputRef"
-    />
+    <div class="tbpro-input">
+      <span v-if="prefix" ref="inputPrefix" class="prefix">{{ prefix }}</span>
+      <input
```

Comment:
Can we class the input? I believe I ran into a bug (in another component) where non-class/id selectors would not be scoped for child components. While this doesn't have any child components with inputs, I'd rather keep it explicit.

Reply:
Of course, done.